src/haddock/clis/cli_analyse.py

```diff
@@ -183,6 +183,26 @@ def update_paths(
     default=False
 )
 
+ap.add_argument(
+    "--mode",
+    help="mode of execution",
+    required=False,
+    type=str,
+    default="local",
+    choices=["local", "batch", "mpi"],
+)
+
+ap.add_argument(
+    "--ncores",
+    help="number of cores to use",
+    required=False,
+    type=int,
+    default=1,
+    # maximum allowed value for ncores is 16
```

Comment:
why is it limited to 16 ?

Reply:
good question..cannot remember a good reason for that, I will put something like 100